typhos/positioner.py

```diff
@@ -255,6 +304,31 @@ def _link_limits_by_limits_attr(self):
         self.ui.low_limit.hide()
         self.ui.high_limit.hide()
 
+    @utils.linked_attribute('moving_attribute', 'ui.moving_indicator', True)
+    def _link_moving(self, signal, widget):
+        """Link the positioner moving indicator with the ui element."""
+        if signal is None:
+            widget.hide()
+            return False
+        else:
```

Comment:
Nitpicky optional style suggestion - remove `else:` and de-indent the following block

Reply:
Ah, right, when I wrote this the first time I didn't have a `return` statement in the first if section, so originally it needed to be like this.

Minimizing indent levels in Python is actually a pretty important thing... Easy to get into trouble if you aren't diligent about it.